### PR TITLE
Add subjects in smaller batches

### PR DESCRIPTION
**Pull request by marten**

### CODE REVIEW

File: app/workers/fill_subject_set_worker.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class FillSubjectSetWorker
```
Comment (marten) [inline app/workers/fill_subject_set_worker.rb]: I don't even know what you mean Hound.

File: app/operations/assignments/create.rb [hunk 2]
```diff
@@ -22,7 +22,7 @@ def execute
         subjects: []
       }
 
-      panoptes.add_subjects_to_subject_set(subject_set["id"], subject_ids)
+      FillSubjectSetWorker.perform_async(subject_set["id"], subject_ids)
```
Comment (shaunanoordin) [inline app/operations/assignments/create.rb]: Query: where is `perform_async` defined? Is this implicit in the Ruby naming - i.e. by adding `_async` it makes the `.perform()` function asynchronous?

Comment (marten) [inline app/operations/assignments/create.rb]: It's a Sidekiq thing: https://github.com/mperham/sidekiq/wiki/Getting-Started

Doesn't work everywhere in Ruby, sadly. Although with a Sidekiq alternative called DelayedJob, you can actually do `whatever.delay.some_method(a, b, c)` and it'll magically work on anything. That magic bites you in the ass at some point though.

Comment (shaunanoordin) [inline app/operations/assignments/create.rb]: OK, that was my only query - PR LGTM!

**Comment by shaunanoordin** [comment]
## Status
LGTM

## Actions
Merging

**Pull request merged**